backstop.config.js

```diff
@@ -0,0 +1,64 @@
+const glob = require("glob")
+const path = require('path');
+const argv = require('minimist')(process.argv.slice(2));
+
+const defaultScenarioOptions = {
+  misMatchThreshold: 0,
+  requireSameDimensions: true,
+  delay: 100
+};
+
+let defaultOptions = {
+  "id": "airship",
+  "viewports": [
+    {
+      "label": "phone",
+      "width": 320,
+      "height": 480
+    },
+    {
+      "label": "tablet",
+      "width": 1024,
+      "height": 768
+    }
+  ],
+  "onBeforeScript": "puppet/onBefore.js",
+  "onReadyScript": "puppet/onReady.js",
+  "paths": {
+    "bitmaps_reference": "backstop_data/bitmaps_reference",
+    "bitmaps_test": "backstop_data/bitmaps_test",
+    "engine_scripts": "backstop_data/engine_scripts",
+    "html_report": "backstop_data/html_report",
+    "ci_report": "backstop_data/ci_report"
+  },
+  "report": ["browser"],
+  "engine": "puppeteer",
+  "engineOptions": {
+    "args": ["--no-sandbox"]
+  },
+  "asyncCaptureLimit": 5,
+  "asyncCompareLimit": 50,
+  "debug": false,
+  "debugWindow": false
+};
+
+const report = {
+  report: argv.report === 'ci'
```

Comment:
we can use the env vars instead arguments

Reply:
:+1: